website_crm_recaptcha/__manifest__.py

```diff
@@ -12,15 +12,16 @@
     "license": "AGPL-3",
     "application": False,
     'installable': True,
+    "uninstall_hook": "uninstall_hook",
     "depends": [
         "website_crm",
         'website_form_recaptcha',
     ],
     "data": [
-        "data/ir_model_data.xml",
-        'views/website_crm_template.xml',
+        "data/ir_model_data.xml",'
+        'te.xml',
```

Comment:
?

Reply:
Oops, some bug in vscode or some extension... Fixed.